src/main/java/frc/robot/subsystems/IntakeSystem.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.AlternateEncoderType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxAlternateEncoder;
+import com.revrobotics.CANSparkMax.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class IntakeSystem extends SubsystemBase {
+  public DutyCycleEncoder intakeEncoder;
```

Comment:
We're not using an absolute encoder, so all references to it should be removed. Also the purpose of the intake should just be taking in and pushing out game pieces, so a setSpeed function should be used.